src/Engine/Manager/TriggerManager.cpp

```diff
@@ -54,7 +54,7 @@ Component::Trigger* TriggerManager::CreateTrigger(const Json::Value& node) {
         repeat->triggerCharges = node.get("triggerCharges", 0).asInt();
         repeat->targetFunction.push_back(node.get("triggerFunction", "").asString());
 
-        repeat->collidedEntityUID = node.get("triggerCollidedEntity", 0).asInt();
+        repeat->collidedEntityUID = node.get("triggerCollidedEntityUID", 0).asInt();
```

Comment:
This is actually a new bug, but was fixed in #874. Should be fine when branch is updated against master.